src/main/java/in/projecteka/gateway/GatewayConfiguration.java

```diff
@@ -195,14 +195,15 @@ public String load(String key) {
     }
 
     @Bean({"bridgeMappings"})
-    public CacheAdapter<Pair<String, ServiceType>, String> createLoadingCacheAdapterForBridgeMappings() {
-        return new LoadingCacheAdapter<>(createMappingCacheForBridge(12));
+    public CacheAdapter<Pair<String, ServiceType>, String> createLoadingCacheAdapterForBridgeMappings(
+            @Value("${gateway.cacheExpiry}") int expiry) {
```

Comment:
Can we rename this to `gateway.bridgeCacheExpiry`? cacheExpiry sounds like for all the caches?

Reply:
Done